GEOSldas_App/GEOSldas_HIST.rc

```diff
@@ -107,46 +117,47 @@ COLLECTIONS:
   tavg24_1d_lnd_Nt.frequency:   240000,
   tavg24_1d_lnd_Nt.ref_time:    000000,
-  tavg24_1d_lnd_Nt.fields:     'WET3'       , 'GridComp'  , 'GWETPROF'     ,
+  tavg24_1d_lnd_Nt.fields:     'GRN'        , 'VEGDYN'  ,
+                               'LAI'        , 'VEGDYN'  ,
+                               'WET3'       , 'GridComp'  , 'GWETPROF'     ,
                                'WET2'       , 'GridComp'  , 'GWETROOT'     ,
                                'WET1'       , 'GridComp'  , 'GWETTOP'      ,
                                'WCPR'       , 'GridComp'  , 'PRMC'         ,
                                'WCRZ'       , 'GridComp'  , 'RZMC'         ,
                                'WCSF'       , 'GridComp'  , 'SFMC'         ,
-                               'TPSNOW'     , 'GridComp'  ,
-                               'TPUNST'     , 'GridComp'  , 'TUNST'        ,
-                               'TPSAT'      , 'GridComp'  , 'TSAT'         ,
-                               'TPWLT'      , 'GridComp'  , 'TWLT'         ,
-                               'TPSURF'     , 'GridComp'  , 'TSURF'        ,
-                               'GRN'        , 'VEGDYN'  ,
-                               'LAI'        , 'VEGDYN'  ,
-                               'TP1'        , 'GridComp'  ,
-                               'TP2'        , 'GridComp'  ,
-                               'TP3'        , 'GridComp'  ,
-                               'TP4'        , 'GridComp'  ,
-                               'TP5'        , 'GridComp'  ,
-                               'TP6'        , 'GridComp'  ,
-                               'PRLAND'     , 'GridComp'  , 'PRECTOTLAND'  ,
-                               'SNOLAND'    , 'GridComp'  , 'PRECSNOLAND'  ,
-                               'TSLAND'     , 'GridComp'  , 'SNOMAS'       ,
-                               'SNOWDP'     , 'GridComp'  , 'SNODP'        ,
-                               'EVPSOI'     , 'GridComp'  , 'EVPSOIL'      ,
-                               'EVPVEG'     , 'GridComp'  , 'EVPTRNS'      ,
-                               'EVPINT'     , 'GridComp'  , 'EVPINTR'      ,
-                               'EVPICE'     , 'GridComp'  , 'EVPSBLN'      ,
-                               'RUNSURF'    , 'GridComp'  , 'RUNOFF'       ,
-                               'BASEFLOW'   , 'GridComp'  ,
+                               'CAPAC'      , 'GridComp'  , 'INTRWATR'     ,
+                               'TPSNOW'     , 'GridComp'  , 'TPSNOWLAND'   ,
+                               'TPUNST'     , 'GridComp'  , 'TUNSTLAND'    ,
+                               'TPSAT'      , 'GridComp'  , 'TSATLAND'     ,
+                               'TPWLT'      , 'GridComp'  , 'TWLTLAND'     ,
+                               'TPSURF'     , 'GridComp'  , 'TSURFLAND'    ,
+                               'TP1'        , 'GridComp'  , 'TSOIL1'       ,         ! CATCH GC: TP1,  ENSAVG GC: TSOIL1TILE    
```

Comment:
TBD:  As of right now, the source variable name depends on the source GridComp.  For single-member simulations, the source GridComp is CATCH and the variable name is TP1.  For ensemble simulations, the source GridComp is ENSAVG and the variable name is TSOIL1TILE.   By changing the variable name TSOIL1TILE to TP1 in GEOS_EnsGridComp.F90, would the source variable name TP1 work regardless of whether the source GridComp is CATCH or ENSAVG GridComp?